CONTRIBUTING.rst

```diff
@@ -18,12 +18,71 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, PEPs that are not marked as Draft or Active are usually considered
+to be historical documents rather than documentation or living specifications.
+To propose a substantially revised version of the original ideas,
+you should generally consider opening a new PEP instead.
+As such, make sure to consider the type and status of the PEP(s)
+in question before proposing changes:
+
+* **Rejected**, **Deferred**, **Withdrawn** and **Superceded** PEPs should
+  usually only be modified when necessary to correct technical issues with
+  header processing or that prevent content from rendering as originally
+  intended.
+
+* **Final** PEPs are considered complete; while small fixes to clearly
+  unintentional errata in the document that meaningfully impair understanding
+  may be considered, substantive changes to the content generally will not.
+
+* **Accepted** PEPs may be updated to reflect minor details of the final
+  implementation, but significant changes to the normative specification
+  should be avoided.
+
+* **Provisional** PEPs can be incrementally modified if needed, though
+  major rewrites and compatibility breakage are discouraged unless absolutely
+  necessary and any changes should be preceded by discussion and general
+  consensus among the authors and the relevant community.
+
+* **Active** PEPs can and should be updated whenever the information described
+  changes. Substantive changes to governance PEPs should be reviewed by the
+  Steering Council (by opening an
+  `issue <https://github.com/python/steering-council/issues>`__ in the SC repo)
+  and are subject to any further conditions stated in the PEP itself.
+
+* **Draft** PEPs are freely open for discussion and proposed modification,
+  at the discretion of the author. Copyediting, formatting and proofreading
+  can be opened here as pull requests, while more substantive content
+  changes should generally be first proposed on the PEP's discussion thread
+  listed in its ``Discussions-To`` header.
```

Comment:
We should also mention that once the PEP has been submitted for pronouncement to the SC or delegate, further changes could delay the process.

Reply:
Good point; I revised this paragraph accordingly